packages/graph/hash_graph/lib/graph/src/provenance/ids.rs

```diff
@@ -0,0 +1,66 @@
+use core::fmt;
+
+use postgres_types::{FromSql, ToSql};
+use serde::{Deserialize, Serialize};
+use utoipa::{openapi::Schema, ToSchema};
+use uuid::Uuid;
+
+#[derive(
+    Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, ToSchema, FromSql, ToSql,
+)]
+#[repr(transparent)]
+#[postgres(transparent)]
+pub struct AccountId(Uuid);
```

Comment:
I don't think this should live in `provenance` this should probably live inside `identifier.rs`. The others are records for provenance and auditing, but this is a lot more foundational and global.